contracts/Liquidator/Liquidator.sol

```diff
@@ -7,13 +7,20 @@ import "@openzeppelin/contracts-upgradeable/access/Ownable2StepUpgradeable.sol";
 import "@venusprotocol/governance-contracts/contracts/Governance/AccessControlledV8.sol";
 import "./LiquidatorStorage.sol";
 
+enum IncomeType {
+    SPREAD,
+    LIQUIDATION
+}
```

Comment:
I feel like this enum belongs somwhere else, not sure where though

Reply:
We can keep it here "today" and import it from the new package in the near future